code/src/components/SingleTodo.js

```diff
@@ -0,0 +1,160 @@
+import React from "react"
+import { useDispatch } from "react-redux"
+import styled from "styled-components"
+
+//--- reducer ---
+import { todos } from "../reducer/todos"
+
+
+export const SingleTodo = ({ id, text, complete }) => {
+
+  const dispatch = useDispatch()
+
+  const handleCheckboxClick = () => {
+    dispatch(todos.actions.toggleComplete(id))
+  }
+
+  const handleRemoveButtonClick = () => {
+    dispatch(todos.actions.removeTask(id))
+  }
+
+  return (
+
+    <TaskContainer>
+      <TextContainer>
+
+        <Label>
+          <Input
+            type="checkbox"
+            checked={complete}
+            onChange={handleCheckboxClick}
+          />
+        </Label>
+
+        {complete ? (<TextCompleted>{text}</TextCompleted>) : (<Text>{text}</Text>)}
+
+      </TextContainer>
+      <div>
+        <Button
+          type="button"
+          onClick={handleRemoveButtonClick}>
+          <Emoji>
+            <span
+              role="img"
+              aria-label="red-cross">
```

Comment:
aria-label should maybe be named different because the cross is grey? Or is the cross supposed to be red?

Reply:
forgot to update that to grey, thanks! :) fixed now